tests/test_component_media.py

```diff
@@ -9,115 +9,47 @@
 from django.utils.html import format_html, html_safe
 from django.utils.safestring import mark_safe
 
-from django_components import Component, registry, types
+from django_components import Component, registry, render_dependencies, types
 
 from .django_test_setup import setup_test_config
 from .testutils import BaseTestCase, autodiscover_with_cleanup
 
-setup_test_config()
+setup_test_config({"autodiscover": False})
 
 
 class InlineComponentTest(BaseTestCase):
     def test_html(self):
         class InlineHTMLComponent(Component):
             template = "<div class='inline'>Hello Inline</div>"
 
-        comp = InlineHTMLComponent("inline_html_component")
         self.assertHTMLEqual(
-            comp.render(Context({})),
+            InlineHTMLComponent.render(),
             "<div class='inline'>Hello Inline</div>",
         )
 
-    def test_html_and_css(self):
-        class HTMLCSSComponent(Component):
-            template = "<div class='html-css-only'>Content</div>"
-            css = ".html-css-only { color: blue; }"
-
-        comp = HTMLCSSComponent("html_css_component")
-        self.assertHTMLEqual(
-            comp.render(Context({})),
-            "<div class='html-css-only'>Content</div>",
-        )
-        self.assertHTMLEqual(
-            comp.render_css_dependencies(),
```

Comment:
I removed / replaced the tests that replied on `Component. render_dependencies` and its variants, since those methods were removed.